src/pages/steps/EventTypeAndThemeStep.tsx

```diff
@@ -210,6 +226,10 @@ const EventTypeAndThemeStep = ({
   const [, hash] = asPath.split('#');
   const eventTypeAndThemeContainer = useRef(null);
 
+  const isCultuurkuurEvent =
+    scope === OfferTypes.EVENTS &&
+    watch('audience.audienceType') === AudienceType.EDUCATION;
```

Comment:
In `LocationStep` we have a [different way to check](https://github.com/cultuurnet/udb3-frontend/blob/main/src/pages/steps/LocationStep.tsx#L252) so I wonder if these should be made to match now that the audience info was moved before the location

Reply:
Yeah that is specific to that step and a weird flow we have there.

A.t.m. you can choose a country or no country (aka cultuurkuur) that is represented as country undefined.
This will no longer be the case